static/app/components/forms/teamSelector.tsx

```diff
@@ -249,12 +251,14 @@ function TeamSelector(props: Props) {
     <SelectControl
       ref={selectRef}
       options={options}
+      onInputChange={debounce(val => void onSearch(val), DEFAULT_DEBOUNCE_DURATION)}
```

Comment:
is the void here just for typing or does debounce do something with the return value of the debounced func?

Reply:
yeah the onInputChange will actually try and render what is returned here, hence the void.

I could also write as `{ ... }`